src/main/scala/rx/lang/scala/observables/AsyncOnSubscribe.scala

```diff
@@ -0,0 +1,120 @@
+package rx.lang.scala.observables
+
+import rx.annotations.Experimental
+import rx.lang.scala.{Notification, Observable, Observer}
+
+/**
+  * An utility class to create observables that start acting when subscribed to and respond correctly
+  * to back pressure requests from subscribers.
+  */
+object AsyncOnSubscribe {
+  /**
+    * Generates an `Observable` that synchronously calls the provided `next` function
```

Comment:
shouldn't this be '**a**synchronously'? (also in other scaladoc sections in this object)